packages/palette/src/elements/Text/Text.shared.tsx

```diff
@@ -14,7 +14,7 @@ import { TextVariant } from "./tokens"
  * Text
  */
 export type TextProps = TypographyProps &
-  ColorProps & {
+  Omit<ColorProps, "color"> & {
     variant?: TextVariant
     textColor?: ResponsiveValue<Color>
```

Comment:
Can we change this to just `color` actually, now that we have removed the one from `ColorProps`?

Reply:
So actually `color` works fine as well. There's a long standing issue with styled-system's `ColorProps` that interferes with the native color prop. We omit it here but it still works as you'd expect.